<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1030</width>
                <height>521</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
                <layout class="QVBoxLayout" name="BaseMainLayout">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QPushButton" name="Base64Button">
                                    <property name="minimumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Base64</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="Base32Button">
                                    <property name="minimumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Base32</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="Base16Button">
                                    <property name="minimumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Base16</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="Base85Button">
                                    <property name="minimumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Base85ASC</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="Base85RFCButton">
                                    <property name="minimumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Base85RFC</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="BaseChangeButtonSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QPushButton" name="BaseEButton">
                                    <property name="minimumSize">
                                        <size>
                                            <width>200</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>200</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>Base64隐写提取</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <item>
                                <widget class="QLabel" name="BaseTableTips">
                                    <property name="text">
                                        <string>编码表:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="BaseTableBox"/>
                            </item>
                            <item>
                                <spacer name="TableSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QPushButton" name="BaseEncButton">
                                    <property name="minimumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>编码</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="BaseDecButton">
                                    <property name="minimumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>120</width>
                                            <height>45</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string>解码</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_5">
                            <item>
                                <layout class="QVBoxLayout" name="verticalLayout">
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                                            <item>
                                                <widget class="QPushButton" name="BaseTextInputButton">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                    <property name="minimumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="maximumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="text">
                                                        <string>打开</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QPushButton" name="BaseTextOutputButton">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                    <property name="minimumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="maximumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="text">
                                                        <string>另存为</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QCheckBox" name="BaseTextEvalCheckBox">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                    <property name="minimumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="maximumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="text">
                                                        <string>启用eval</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QCheckBox" name="BaseDoNotLoadFileCheckBox">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                    <property name="minimumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="maximumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="text">
                                                        <string>大文件</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <widget class="QTextEdit" name="BaseTextBox"/>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <layout class="QVBoxLayout" name="verticalLayout_2">
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                                            <item>
                                                <widget class="QPushButton" name="BaseCipherInputButton">
                                                    <property name="minimumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="maximumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="text">
                                                        <string>打开</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QPushButton" name="BaseCipherOutputButton">
                                                    <property name="minimumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="maximumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="text">
                                                        <string>另存为</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QPushButton" name="BaseTranslateButton">
                                                    <property name="minimumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="maximumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="text">
                                                        <string>交换</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QCheckBox" name="BaseAutoLoadCheckBox">
                                                    <property name="minimumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="maximumSize">
                                                        <size>
                                                            <width>120</width>
                                                            <height>45</height>
                                                        </size>
                                                    </property>
                                                    <property name="text">
                                                        <string>AutoLoad</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <widget class="QTextEdit" name="BaseCipherBox"/>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
